src/test/java/io/strimzi/kafka/bridge/http/AdminClientTest.java

```diff
@@ -66,11 +67,11 @@ void getTopicTest(VertxTestContext context) throws Exception {
                         for (int i = 0; i < 2; i++) {
                             JsonObject partition = partitions.getJsonObject(i);
                             assertThat(partition.getInteger("partition"), is(i));
-                            assertThat(partition.getInteger("leader"), is(1));
+                            assertThat(partition.getInteger("leader"), is(0));
                             JsonArray replicas = partition.getJsonArray("replicas");
                             assertThat(replicas.size(), is(1));
                             JsonObject replica0 = replicas.getJsonObject(0);
-                            assertThat(replica0.getInteger("broker"), is(1));
+                            assertThat(replica0.getInteger("broker"), is(0));
```

Comment:
why this change? the test was failing?

Reply:
Correct.

Reply:
Really? I rebuilt the bridge a lot of times during these days for release without getting any error ...